### PR TITLE
MINOR: Default to 5 partitions of the __consumer_offsets topic in Streams integration tests

### PR DESCRIPTION
**Pull request by stanislavkozlovski**
Given that the tests do not create clusters larger than 3, we do not gain much by creating 50 partitions for that topic. Reducing it should slightly increase test startup and shutdown speed

### CODE REVIEW

**Comment by stanislavkozlovski** [comment]
For reference, we had done the same optimization in Kafka's integrations tests here - https://github.com/apache/kafka/commit/b62f156eb480498eba338b1746a995ed0d8ce871

I see Streams already defaults GroupInitialRebalanceDelayMsProp to 0

**Review by mjsax — APPROVED** [review]
LGTM.

Call for second review @guozhangwang @bbejeck @vvcephei @ableegoldman @cadonna @abbccdda

**Review by bbejeck — APPROVED** [review]
LGMT.  Thanks @stanislavkozlovski! 

Waiting for green build to merge.

**Comment by bbejeck** [comment]
All build timed out.

retest this please

**Comment by mjsax** [comment]
Java 11 / 2.12 timeout out.

Java 11 / 2.13: `kafka.admin.ReassignPartitionsClusterTest.shouldPerformMultipleReassignmentOperationsOverVariousTopics`

Java 8:
```
kafka.api.CustomQuotaCallbackTest.testCustomQuotaCallback
kafka.api.SaslSslAdminClientIntegrationTest.testLogStartOffsetAfterDeleteRecords
```

Retest this please.

**Comment by bbejeck** [comment]
Java 11/2.13 failed test results cleaned up
Java 11/2.12 and Java 8/2.11 passed 

retest this please

**Review by guozhangwang — APPROVED** [review]

**Pull request merged**
